docs/ant-automatic-installers.md

```diff
@@ -0,0 +1,78 @@
+# Ant Automatic Installers
+
+Choose an automatic installer to let Jenkins install this tool for you on demand.
+If you check this option, you'll then be asked to configure a series of "installer"s for this tool, where each installer defines how Jenkins will try to install this tool.
+
+For a platform-independent tool (such as Ant), configuring multiple installers for a single tool does not make much sense, but for a platform dependent tool, multiple installer configurations allow you to run a different set up script depending on the agent environment.
+
+## Install from Apache
+
+This installs Ant from [apache.org](https://ant.apache.org/) using one of the publicly available versions.
+
+### Configuration Options
+
+* Version: the ant version to install from apache.org.
+
+## Extract \*.zip/\*.tar.gz
+
+Downloads a tool archive and installs it within Jenkins's working directory. Example: http://apache.promopeddler.com/ant/binaries/apache-ant-1.7.1-bin.zip (or whatever mirror is closest to your server) and specify a subdir of apache-ant-1.7.1.
+
+### Configuration options
+
+* Label - a label for this installer
+* Download URL for binary archive - URL do download the binary archive from
+* Subdirectory of extracted archive - directory in the extracted archive containing the tools to install
+
+## Run Batch Command
+
+Runs a shell command of your choice to install the tool. Ubuntu example, assuming the Jenkins user is in /etc/sudoers:
+
+```sh
+sudo apt-get --yes install openjdk-6-jdk
```

Comment:
Either rewrite to actually talk about Ant, or delete most of this content and just link to general references on jenkins.io.